<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deleteBox</class>
 <widget class="QDialog" name="deleteBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources/ressources.qrc">
    <normaloff>:/icons/icons/rddzscraper-24.png</normaloff>:/icons/icons/rddzscraper-24.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="columnComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Strings</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QRadioButton" name="stringContainingRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>containing</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">filterButtonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="stringNotContainingRadioButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>not containing</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">filterButtonGroup</string>
      </attribute>
     </widget>
     <widget class="QCheckBox" name="useRegexCheckBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>use regex</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Numbers</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QRadioButton" name="numberLowerRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>lower</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">filterButtonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="numberHigherRadioButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>higher</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">filterButtonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="numberEqualRadioButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>equal</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">filterButtonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="numberNotEqualRadioButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>different</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">filterButtonGroup</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputValue"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources/ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>deleteBox</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>deleteBox</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="filterButtonGroup"/>
 </buttongroups>
</ui>
